### PR TITLE
xml_variable_mp into xml_variable

### PR DESCRIPTION
**Pull request by Setnea**
Introducing following features:
a) class VariableMP
b) TASK_IF modyfications

### CODE REVIEW

File: python/modules/variable.py [hunk 1]
```diff
@@ -89,53 +89,122 @@ def get_value(self, env, args = {}):
...
-        names = list(mp_dict[key_0].keys()) 
+        key_0 = list(self.mp_dict.keys())[0]
+        names = list(self.mp_dict[key_0].keys()) 
         #if we assume that number of information between wells is constant
         return names
 
-    def get_well_params(mp_dict, well):
+    def get_well_params(self, well):
         """ 
         Gives all parameters values for a given well
```
Comment (stork119) [inline python/modules/variable.py]: What is a structure of returned object : dictionary, list, variable ?

File: python/modules/variable.py [hunk 2]
```diff
@@ -89,53 +89,122 @@ def get_value(self, env, args = {}):
...
+    b)
+    Parameters with type 'map_plate', referring to chosen map_plate element
+    are represented in  xml settings as following example:
+    <parameter key = "<KEY>" value/mp_name = <MP_NAME> type = "map_plate">
+        <parameter key = "well" value = "mp_key" type = "ref">
+        <parameter key = "param" value = "stimulation.1.1">
+    </parameter>
+    """
     def __init__(self, key, value, args = {}):
         Variable.__init__(self, key, value, args = {})
```
Comment (stork119) [inline python/modules/variable.py]: http://stackoverflow.com/questions/400739/what-does-asterisk-mean-in-python

**Review by stork119 — APPROVED** [review]
Add some more precise comments

File: python/modules/variable.py (hunk 1 above)
Comment (Setnea) [reply python/modules/variable.py]: List. It's inner function. I added comments describing returned value to map_plate functions.

**Pull request merged**